_posts/2016-10-07-dream-of-bemifornication.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: "Dream of BEMifornication - or how we refactored our scss with linting"
+date: 2016-10-07
+author: axlwaii
+comments: true
+tags:
+  - CSS
+  - SCSS
+  - refactoring
+---
+
+
+Writing a lot of code over the years also means to deal with refactoring at some point. Your skills improve over time and technologies envolve. Coming back to an old project often makes it hard to understand what the heck younger you was thinking when writing and structuring this code. At least that’s the way I sometimes feel. 
+
+For example when I started learning SASS I felt in love with nesting everything. Back then I tought nesting elements would be the best way to avoid css conflicts … I figured out that might just be true up to a certain level of nesting. [BEM](http://getbem.com/) came around and we started sticking to its convention, leaving previously written css code mostly untouched. Things started to get messy in our frontend codebase.
```

Comment:
comma after "learning SASS"

s/tought/thought/

Suggestion: "When BEM came around, we started …"